docs/admin/dns.md

```diff
@@ -182,6 +182,111 @@ Address 1: 10.0.0.1
 
 If you see that, DNS is working correctly.
 
+### Troubleshooting tips.
+If above `nslookup` command does not work, here are some hints:
+
+#### Check the local DNS configuration first
+Take a look inside the resolv.conf file. (See "Inheriting DNS from the node" and "Known issues" sections for more information)
```

Comment:
These two sections are currently in [kubernetes/build/kube-dns/README.md](https://github.com/kubernetes/kubernetes/blob/master/build/kube-dns/README.md), but are being removed in kubernetes/kubernetes#32931.

Don't know why they are not included in previous PR. So I moved them here as well.